analysis/queries.py

```diff
@@ -21,6 +21,18 @@
 from database import utils as db_utils
 
 
+def get_experiment_benchmarks(experiment_names):
+    """Get benchmarks of |experiment_names| from the database."""
+    with db_utils.session_scope() as session:
+        snapshots_query = session.query(Trial.benchmark)\
+            .select_from(Experiment)\
+            .join(Trial)\
+            .filter(Experiment.name.in_(experiment_names))\
+            .filter(Trial.preempted.is_(False))
+
+    return pd.read_sql_query(snapshots_query.statement, db_utils.engine)
```

Comment:
make this return `result['benchmark'].unique().tolist()` so that it returns a list of unique benchmark names, which is what the name of this function implies